src/test/scala/ai/privado/languageEngine/ruby/download/ExternalDependenciesDownloadTest.scala

```diff
@@ -0,0 +1,55 @@
+package ai.privado.languageEngine.ruby.download
+
+import ai.privado.cache.RuleCache
+import ai.privado.languageEngine.ruby.cache.PackageTable
+import better.files.File
+import io.joern.rubysrc2cpg.{Config, RubySrc2Cpg}
+import io.joern.x2cpg.X2Cpg.applyDefaultOverlays
+import io.shiftleft.codepropertygraph.generated.Cpg
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import scala.collection.mutable
+
+class ExternalDependenciesDownloadTest extends ExternalDependenciesDownloadTestBase {
+
+  override val rubyFileContentMap: Map[String, String] = getContent()
+
+  def getContent(): Map[String, String] = {
+    val testGemMap = mutable.HashMap[String, String]()
+
+    val gemfileContent =
```

Comment:
Write similar test case for some popular ruby gem

Reply:
okay sure will add more test case.